### PR TITLE
[backport/1.22] repo: Release `1.22.7`

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: docs/root/version_history/current.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-1.22.7 (Pending)
-================
+1.22.7 (February 8, 2022)
+=========================
```
Comment (phlax) [inline docs/root/version_history/current.rst:3]: changelogs!!!

**Comment by phlax** [comment]
not much has actually changed on this branch so the only changelog currently is the distroless image update

https://storage.googleapis.com/envoy-pr/814da86/docs/version_history/current.html

**Comment by phlax** [comment]
cc @ggreenway

**Review by ggreenway — APPROVED** [review]

**Pull request merged**